generators/app/index.js

```diff
@@ -9,27 +9,35 @@ var semver = require('semver');
 var versions = require('./dependency-versions.js');
 var yosay = require('yosay');
 var _ = require('lodash');
+var genConfig = null;
 
 module.exports = yeoman.Base.extend({
+  _getConfigValue: function(key) {
+    var configValue = this.config.get(key);
+    var defaultValue = genConfig[key];
```

Comment:
Since ```getConfig``` can be null/undefined shouldn't you have some sort of guard against dereferencing null/undefined?